dev-libs/libbase58/libbase58-0.1.4.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A library for encoding/decoding base58"
+DESCRIPTION="libbase58 provides encoding and decoding for the base58 format, \
+used in Bitcoin addresses."
+HOMEPAGE="https://github.com/bitcoin/libbase58"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libbase58/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="97e8c3387fd4ed90208a8101e658743efbb6e21714e594f0828e6414f92c0b1d"
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libbase58$secondaryArchSuffix = $portVersion
+	lib:libbase58$secondaryArchSuffix = 0.0.2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libbase58${secondaryArchSuffix}_devel = $portVersion
+	devel:libbase58$secondaryArchSuffix = 0.0.2
```

Comment:
This change is neutral, but it can be handy, since it allows to have the "0.0.2" appear only once in the recipe.
```diff
-	devel:libbase58$secondaryArchSuffix = 0.0.2
+	devel:libbase58$secondaryArchSuffix = $libVersion
```